use-regex-1.4.patch

```diff
@@ -0,0 +1,164 @@
+From 0509d49c4bd5931a78b4f3df5771cdd2d369e238 Mon Sep 17 00:00:00 2001
```

Comment:
Nitpick: You can put those patches in a directory to keep the repository clean once created.